### PR TITLE
Events pages for May 2022

### PR DESCRIPTION
**Pull request by krisfreedain**
Signed-off-by: Kris Freedain <freedak@amazon.com>

### Description
New events pages for May 2022
 
### Issues Resolved
N/A

### Check List
- [ x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _events/2022-0503.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+tz: UTC -7
+title: OpenSearch Community Meeting - 2022-05-03
+online: true
+signup:
+    url: https://www.meetup.com/OpenSearch/events/285396685
+    title: Join on Meetup
+
+---
+
+Join us for an open office community meeting.
```
Comment (stockholmux) [inline _events/2022-0503.markdown:13]: Maybe you want to indicate the open Q&A in the title?

File: _events/2022-0509.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tz: UTC -7
+title: OpenSearch Community Meeting - 2022-05-09
+online: true
+signup:
+    url: https://www.meetup.com/OpenSearch/events/285396734/
+    title: Join on Meetup
+
+---
+
+Join us for our weekly online community meeting.
```
Comment (stockholmux) [inline _events/2022-0509.markdown:13]: I might change the 'weekly' to be more clear that this format isn't weekly

File: _events/2022-0503.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Meeting topics and passcode are listed in the [collaborative agenda](https://hackmd.io/@HmdZWaVnQU6M8icdvC5TwQ/HyJ300V4c) on HackMD.
```
Comment (stockholmux) [inline _events/2022-0503.markdown:15]: Do you want to do agendas for open Q&A?

**Review by stockholmux — COMMENTED** [review]
Up to you, but personally, I would make the Q&A meetings and the programmed community meetings more distinct.

File: _events/2022-0503.markdown (hunk 3 above)
Comment (krisfreedain) [reply _events/2022-0503.markdown:15]: made a suitable change to address this.

**Review by krisfreedain — COMMENTED** [review]
made suitable changes to address the comments - let me know. thanks

File: _events/2022-0509.markdown (hunk 2 above)
Comment (krisfreedain) [reply _events/2022-0509.markdown]: made a suitable change to address this.

File: _events/2022-0503.markdown (hunk 1 above)
Comment (krisfreedain) [reply _events/2022-0503.markdown]: made a suitable change to address this.

**Comment by krisfreedain** [comment]
thanks @stockholmux - made some changes that will work

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**